meinberlin/assets/scss/components/_map.scss

```diff
@@ -16,8 +16,8 @@
     flex-direction: column;
     justify-content: space-between;
 
-    margin: 0.5em;
-    width: 14.5em;
+    padding: $padding;
+    width: 14.5rem;
 }
```

Comment:
is this ok? couldn't think of where else we used map popups so thought this would be a fine solution?